core/code/ornaments.js

```diff
@@ -71,5 +139,109 @@ window.ornaments = {
       });
       delete this._portals[guid];
     }
+  },
+  initOrnaments: function () {
+    this.knownOrnaments = {};
+    this.save();
+  },
+
+  load: function () {
+    var dataStr;
+    try {
+      dataStr = localStorage['excludedOrnaments'];
```

Comment:
```suggestion
      dataStr = localStorage.getItem('excludedOrnaments');
```
geItem/setItem are the standarized way to access localStorage.
(note: getItem will return `null` if value doesn't exists)

anyway, `localStorage['excludedOrnaments'];` will also work fine.

Reply:
implemented